pkg/diff/chunk.go

```diff
@@ -516,19 +516,18 @@ func getSplitFields(table *model.TableInfo, splitFields []string) ([]*model.Colu
 	return cols, nil
 }
 
-// CheckJob is the struct of job for check
-type CheckJob struct {
-	Schema string
-	Table  string
-	Where  string
-	Args   []string
-}
+// GetChunks gets some chunks.
+func GetChunks(ctx context.Context, table *TableInstance, splitFields, limits string, chunkSize int, collation string, useTiDBStatsInfo bool) (chunks []*ChunkRange, fromCheckpoint bool, err error) {
+	chunks, err = loadChunksInfo(ctx, table.Conn, table.InstanceID, table.Schema, table.Table)
```

Comment:
what if fail to save some chunks?

Reply:
will recheck these chunks

Reply:
would populate chunk again? I understand the progress is: populate chunk/load chunk -> save chunk -> check chunk

Reply:
in fact the progress is :  populate chunk -> save chunk -> check chunk -> update chunk

Reply:
if meet some error when save chunks ,the table's state will be notChecked, and when diff restart, will clean checkpoint and  populate chunk again